apps/core/serializers.py

```diff
@@ -110,3 +111,57 @@ def validate(self, attrs):
             return None
         data = super(TokenPairSerializer, self).validate(attrs)
         return data
+
+
+class PasswordResetSerializer(serializers.ModelSerializer):
+    email = serializers.EmailField(
+        required=True,
+    )
+
+    class Meta:
+        model = User
+        fields = ("email", )
+
+    def validate(self, attrs):
+        email = attrs["email"]
+        user_model = get_user_model()
+
+        try:
+            user = user_model.objects.get(email=email)
+            attrs["user"] = user
+        except user_model.DoesNotExist:
+            raise serializers.ValidationError(
+                {"email": "There are no users with that Email address!"}
+            )
+
+        if not user.is_email_verified:
+            raise serializers.ValidationError(
+                {"email": "Email is not verified!"}
+            )
```

Comment:
Let's consider that a user for some reason can't or doesn't want to verify his email but he forgot his password.
I don't think we should limit him with his ability to reset the password